packages/runtime-core/src/componentProxy.ts

```diff
@@ -158,10 +158,11 @@ export const PublicInstanceProxyHandlers: ProxyHandler<any> = {
   },
 
   set(target: ComponentInternalInstance, key: string, value: any): boolean {
-    const { data, renderContext } = target
-    if (data !== EMPTY_OBJ && hasOwn(data, key)) {
+    const { data, renderContext, accessCache } = target
+    const n = accessCache![key]
```

Comment:
`n` is not the best variable name, maybe `accessType`?

Reply:
Yep, but I just copy  this. https://github.com/vuejs/vue-next/blob/e683cc6ed4d7b84ce557229f3aa119e11243345f/packages/runtime-core/src/componentProxy.ts#L98
Perhaps we should fix it too.